### PR TITLE
chore: release 2.0.0

### PR DESCRIPTION
**Pull request by hugocaillard**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+#### [2.0.0](https://github.com/hirosystems/clarinet/compare/v1.8.0...v2.0.0) (2023-10-26)
+
+##### New Features
+
+*  Introduce global ~/.clarinet/clarinetrc.toml (#1208) (bbe26ccf)
```
Comment (tippenein) [inline CHANGELOG.md:5]: currently it looks for `~/.clarinet/Settings.toml`

Comment (hugocaillard) [reply CHANGELOG.md:5]: hey @tippenein 
I mentioned in this follow up PR as well: https://github.com/hirosystems/clarinet/pull/1225

Comment (tippenein) [reply CHANGELOG.md:5]: Ah! I see you responded here: https://github.com/hirosystems/clarinet/pull/1225

Thanks!

**Review by lgalabru — APPROVED** [review]
😍

**Pull request merged**